.github/workflows/clp-package-image-build.yaml

```diff
@@ -0,0 +1,103 @@
+name: "clp-package-image-build"
+
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  DEPS_IMAGE_NAME_PREFIX: "clp-core-dependencies-x86-"
```

Comment:
(Similar to the comment at https://github.com/y-scope/clp/pull/1166/files#r2256441706)

I see how it can be awkward to build the package with a "core" dependencies container, but i haven't seen any quicker way to build the package. if there's no significant benefit in differentiating between "core" and "package" dependencies, shall we just unify them (simply rename the existing build env containers).

Reply:
@coderabbitai create an issue to track the effort of creating a universal build env image